en/console-and-shells/completion-shell.rst

```diff
@@ -54,14 +54,47 @@ For example::
 
 Returns::
 
-    --help -h --verbose -v --quiet -q --connection -c --template -t
+    --help -h --verbose -v --quiet -q --everything --connection -c --force -f --plugin -p --prefix --theme -t
+Vous pouvez passer un autre argument représentant une sous-commande du shell :
+cela vous retournera les options spécifiques à cette sous-commande.
 
-Bash Example
-============
+You can also pass an additional argument being the shell sub-command : it will
+output the specific options of this sub-command.
+
+How to enable Bash autocompletion for the CakePHP Console
+=========================================================
+
+Using a Debian distribution
+---------------------------
+
+First, make sure the **bash-completion** library is installed.
+Create a file named **cake** in **/etc/bash_completion.d/** and put the
+:ref:`bash-completion-file-content` inside it.
+
+Save the file, then restart your console.
+
+Using Mac OS X
+--------------
+
+First, make sure the **bash-completion** library is installed.
+You can install it using **homebrew** with the command
+``brew install bash-completion``.
+Create a file named **cake** in **/usr/local/etc/bash_completion.d/** and put
+the :ref:`bash-completion-file-content` inside it.
 
-The following bash example comes from the original author::
+Save the file, then restart your console.
```

Comment:
The only way I thought this was good was in case people want to contribute and add others system instructions.

But I'm on board. Basically, instructions are the same, it's just that the process to install the completion lib and the target directory for the file are different.

I'll make the instructions only for a debian distribution and add a ``.. note::`` about the specifity of OSX.